### PR TITLE
Add windows install test

### PR DESCRIPTION
**Pull request by tannoa2**

### CODE REVIEW

**Comment by tannoa2** [comment]
@keedya @jlongever @uppalk1 @zyoung51 @amymullins

File: test/tests/api/v1_1/os_install_tests.py [hunk 1]
```diff
@@ -177,6 +177,30 @@ def install_ubuntu(self, version, nodes=[], options=None):
...
             }
         self.__post_workflow(graph_name, nodes, body)
+    
+    def install_windowsServer2012(self, version, nodes=[], options=None):
+        graph_name = 'Graph.InstallWindowsServer'
+        os_repo = defaults.get('RACKHD_WINDOWS_REPO_PATH', \
+                               self.__base + '\\isofarm\\windowsServer2012')
+        body = options
+        if body == None:
+        # The value of the productkey below is not a valid product key. It is a KMS client key that was generated to run the workflows without requiring a real product key. This key is
```
Comment (yyscamper) [inline test/tests/api/v1_1/os_install_tests.py]: This line is too long, suggest splitting it into 2 lines.

**Comment by yyscamper** [comment]
:+1:

**Comment by anhou** [comment]
:+1:

**Comment by keedya** [comment]
:+1:

File: test/tests/api/v1_1/os_install_tests.py [hunk 2]
```diff
@@ -177,6 +178,33 @@ def install_ubuntu(self, version, nodes=[], options=None):
                 }
             }
         self.__post_workflow(graph_name, nodes, body)
+    
+    def install_windowsServer2012(self, version, nodes=[], options=None):
+        graph_name = 'Graph.InstallWindowsServer'
+        os_repo = defaults.get('RACKHD_SMB_WINDOWS_REPO_PATH', none)
+        if none == os_repo:
```
Comment (jlongever) [inline test/tests/api/v1_1/os_install_tests.py]: Should be `None`.

**Comment by tannoa2** [comment]
Test this please

**Comment by tannoa2** [comment]
Test this please

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Comment by tannoa2** [comment]
test this please

**Pull request merged**